pkgs/development/libraries/spandsp/default.nix

```diff
@@ -1,23 +1,24 @@
-{stdenv, fetchurl, audiofile, libtiff}:
+{ stdenv, fetchFromGitHub, audiofile, libtiff, autoreconfHook }:
 stdenv.mkDerivation rec {
-  version = "0.0.6";
+  version = "3.0.0";
```

Comment:
OpenWRT folks decided to package this as [separate package](https://github.com/openwrt/telephony/pull/563), maybe we should too?

Reply:
Maybe. I don't remember what else uses this and generally we try to avoid multiple versions of libraries, but in this case it seems reasonable to duplicate.

Reply:
I checked again. The only other things using `spandsp` are:
- pkgs/development/libraries/gstreamer/bad/default.nix
- pkgs/applications/networking/instant-messengers/baresip/default.nix

Which both seem to compile fine with this update. So I'd actually rather *not* duplicate this.